wemake_python_styleguide/visitors/ast/builtins.py

```diff
@@ -125,6 +124,128 @@ def _check_modulo_patterns(
             self.add_violation(consistency.ModuloStringFormatViolation(node))
 
 
+@final
+class WrongFormatStringVisitor(base.BaseNodeVisitor):
+    """Restricts usage of ``f`` strings."""
+
+    _valid_format_index: ClassVar[AnyNodes] = (
+        ast.Str,
```

Comment:
Do we allow `Bytes`? Use this alias: https://github.com/wemake-services/wemake-python-styleguide/blob/master/wemake_python_styleguide/compat/aliases.py#L16